### ISSUE TITLE
Guided Tours: Tour is stuck on preview prompt after using keyboard to exit site preview

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/wp-calypso#5812]
The Guided Tour is stuck on the preview prompt if you use your keyboard's **esc** key to exit the site preview.

#### Steps to reproduce
1. Starting at URL: https://wordpress.com/?tour=main
2. Follow each of the Guided Tour prompts until you reach the **Preview: Click here to see what your site looks like.** prompt. 
3. Preview your site. The Guided Tour prompt will then read as follows: **Take a look at your site—and then close the site preview. You can come back here anytime.**
4. When you have finished previewing your site, hit the keyboard's **esc** key to exit.

#### What I expected

I expected to be able to continue with the Guided Tour and for the next prompt to appear.

#### What happened instead

The **Take a look at your site—and then close the site preview. You can come back here anytime.** prompt remained in the upper left hand corner of my screen and I was unable to dismiss it.

Some people are only able to navigate the web using their keyboard and so it would be good to account for this.

#### Browser / OS version

Chrome / OS X El Capitan V 10.11.5

#### Screenshot / Video

![guided-tours](https://cloud.githubusercontent.com/assets/2998162/15795457/9cc741d2-29e9-11e6-9475-4ad1315fb25c.gif)
